make-nagios-commands-cfg.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+import os
+
+from jinja2 import Environment, FileSystemLoader, Template
+
+smtp_server = os.getenv('SMTP_SERVER', 'localhost')
```

Comment:
Taking for example the [MySQL image](https://github.com/sclorg/mysql-container/tree/master/5.5):

```console
$ docker run --rm -it \
    -e MYSQL_USER=user \
    -e MYSQL_PASSWORD=pass \
    -e MYSQL_DATABASE=db \
    -p 3306:3306 \
    openshift/mysql-55-centos7
...
160714  6:34:24 [Note] /opt/rh/mysql55/root/usr/libexec/mysqld: ready for connections.
Version: '5.5.45'  socket: '/var/lib/mysql/mysql.sock'  port: 3306  MySQL Community Server (GPL)
```

But without the configuration:

```console
$ docker run --rm -it -p 3306:3306 openshift/mysql-55-centos7
You must either specify the following environment variables:
  MYSQL_USER (regex: '^[a-zA-Z0-9_]+$')
  MYSQL_PASSWORD (regex: '^[a-zA-Z0-9_~!@#$%^&*()-=<>,.?;:|]+$')
  MYSQL_DATABASE (regex: '^[a-zA-Z0-9_]+$')
Or the following environment variable:
  MYSQL_ROOT_PASSWORD (regex: '^[a-zA-Z0-9_~!@#$%^&*()-=<>,.?;:|]+$')
Or both.
Optional Settings:
...
```

We may pass in dummy config, but the image itself never make those choices itself.
Doing so, how will users of the image realize they won't get any emails when things go wrong?!